src/main/java/uk/gov/justice/hmpps/datacompliance/services/client/elite2api/Elite2ApiClient.java

```diff
@@ -40,7 +39,6 @@ public OffenderNumbersResponse getOffenderNumbers(final long offset, final long
                 .uri(dataComplianceProperties.getElite2ApiBaseUrl() + OFFENDER_IDS_PATH)
                 .header("Page-Offset", String.valueOf(offset))
                 .header("Page-Limit", String.valueOf(limit))
-                .attributes(clientRegistrationId("dps-data-compliance"))
```

Comment:
In the last PR, WebConfig sets this as a default so no longer need to provide it here